pkg/app/pipedv1/plugin/wait/config/options_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	config "github.com/pipe-cd/pipecd/pkg/configv1"
```

Comment:
This should not be here, isn't it?

Reply:
@khanhtc1202 
Thanks, I fixed them: [2be9e5d](https://github.com/pipe-cd/pipecd/pull/5456/commits/2be9e5dfec5154d34036c0cc5e61b46fcce73bf7)